src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -74,6 +77,28 @@ def create_app():
         # Set app logger to the only one RotatingFileHandler to avoid duplicate logs
         app.logger.handlers = [handler]
 
+        def initialize_created_by_info():
+            from promptflow._sdk._configuration import Configuration
+            from promptflow._sdk._utils import extract_workspace_triad_from_trace_provider
+
+            trace_provider = Configuration.get_instance().get_trace_provider()
+            if trace_provider is None or extract_workspace_triad_from_trace_provider(trace_provider) is None:
+                return
+            try:
+                import jwt
```

Comment:
for get token, consider `promptflow.azure._utils.gerneral.get_token?`

Reply:
Fixed. It will be better if could return `decoded_token` instead...